pkg/publicapi/client/v2/client.go

```diff
@@ -482,6 +482,14 @@ type AuthenticatingClient struct {
 	// Authenticate will be called when the system should run an authentication
 	// flow using the passed Auth API.
 	Authenticate func(context.Context, *Auth) (*apimodels.HTTPCredential, error)
+
+	// NewAuthenticationFlowEnabled is true when new auth flow is detected.
+	// This is kept here for backward compatibility reasons only
+	NewAuthenticationFlowEnabled bool
+
+	// SkipAuthentication is true when new auth flow is detected.
+	// This is kept here for backward compatibility reasons only
+	SkipAuthentication bool
```

Comment:
these are a bit confusing. Do you need two booleans? Can the names be more accurate?

Reply:
TODO: Fix comments